app/views/claims/schools/claims/index.html.erb

```diff
@@ -35,7 +35,7 @@
                 <% end %>
               </ul>
             <% end %>
-            <% row.with_cell(text: l(claim.created_at.to_date, format: :short)) %>
+            <% row.with_cell(text: l(claim.submitted_at.to_date, format: :short)) %>
```

Comment:
To follow the law of Demeter, could we add this to the model?

```ruby
class Claim
  def submitted_on
    submitted_at&.to_date
  end
end
```

```suggestion
            <% row.with_cell(text: l(claim.submitted_on, format: :short)) %>
```

This also handles `nil` calls.

There's also something I'll note, the `l` method doesn't like `nil` values being passed in. I think we only show records _with_ a `submitted_at` value, but I thought I'd mention it.

Reply:
@CatalinVoineag I edited the above _slightly_.